pkg/events/plugin.go

```diff
@@ -51,18 +51,6 @@ type Plugin interface {
 	DeliveryRequest(connID string, sub *fftypes.Subscription, event *fftypes.EventDelivery, data fftypes.DataArray) error
 }
 
-// ChangeEventListener is an optional interface for delivering database change events, only supported for ephemeral connections
-type ChangeEventListener interface {
-	// ChangeEvent delivers a change event - no ack for these
-	ChangeEvent(connID string, ce *fftypes.ChangeEvent)
-}
```

Comment:
So `ChangeEventListeners` are not a distinct concept anymore? `ChangeEvents` are just a thing that you get on the admin websocket now?

Reply:
Mixing together application level events, and these change events, on the same WebSocket with different payloads has been removed. It felt really confusing, and the danger of opening a firehose you can't cope with is now pretty self-evident.

So I guess I think of it as more distinct now, in that you have a whole separate WebSocket to listen to.

So it's got nothing to do with the `events` plugin at all - it's a system-level thing built into the core.